content/08-cardano-sidechains/03-example-evm-sidechain/05-transferring-tokens.mdx

```diff
@@ -0,0 +1,211 @@
+    }
+  }
+}
+```
+
+To obtain the `txIndex` matching the recipient and amount used to lock tokens, proceed as follows.
+
+- Wait until the epochPhase changes to `handover` of epoch `N` or `N+1` and obtain the Merkle proof for your lock action:
+
+```shell
+curl -L -X POST -H 'Content-type:application/json' -d '{
+  "jsonrpc": "2.0",
+  "method": "sidechain_getOutgoingTxMerkleProof",
+  "params": [<N or N+1>, <txIndex>],
+  "id": 1
+  }' https://faucet.sidechain.evmtestnet.iohkdev.io/ | jq
+```
+
+From the return you need to obtain the `proof/bytes` and remove the `0x` that is prepended. Call that string `merkleProof`.
+
+- Wait until the epoch changes and the committee handover has happened. The committee handover happens automatically on the testnets by a service (relay) that tries every minute to find epoch signatures, and succeeds after a new epoch has started and the signatures are obtained. On mainnet, the committee handover will be done by anyone that wants, where there will be an incentive to do so. A good way to verify is by waiting until you observe your transaction in the list of signatures to upload:
+
+```
+curl -L -X POST -H 'Content-type:application/json' -d '{
+    "jsonrpc": "2.0",
+    "method": "sidechain_getSignaturesToUpload",
+    "params": [<limit>],
+    "id": 1
+    }' https://faucet.sidechain.evmtestnet.iohkdev.io/ | jq
+```
+
+Where `limit` = min(limit, number of epochs that are not yet relayed). The default value is `100`, but you can set it to any other number. In a normal case, when the handover happens at every epoch, it should only show 0 or 1 epochs (the last one). Once you identify that there are no roothashes pending in the list under the epoch that your lock was processed (`N` or `N+1`), you can claim your tokens. By now the epoch will be `N+1 `or `N+2`, depending on the phase you performed the lock operation.
+
+- Finally, claim the tokens by submitting a transaction on the main chain with the `merkleProof` you obtained using the **sidechain-cli**: 
+
+```shell
+./sidechain-cli claim-SC_Token --sc-url https://faucet.sidechain.evmtestnet.iohkdev.io/ --signing-key-file /PATH/TO/payment.skey --combined-proof <merkleProof without 0x>
+```
+
+Example output: 
+
+```
+{
+  "endpoint": "ClaimAct",
+  "transactionId": "1334b3dab421911af68b9393e5cc4756c46c9ab1ac567a57450597e174351a48"
+}
+```
+
+Verify amount you locked in your account:
+
+```
+cardano-cli query utxo --testnet-magic 7 --address $MC_ADDR
+```
+
+🎉 Congratulations! At this point, you have successfully claimed your `SC_Token` on the main chain.
+
+# Transfering tokens from the main chain to a sidechain
+
+### Step 01 - Burn SC_Token
+
+To transfer `SC_Token` from the main chain to a sidechain, use the **sidechain-cli** to burn main chain `SC_Token`.
+
+```shell
+./sidechain-cli burn-SC_Token
+  --signing-key-file payment.skey \
+  --sc-url https://faucet.sidechain.evmtestnet.iohkdev.io/ \
+  --recipient ae3dffee97f92db0201d11cb8877c89738353bce \
```

Comment:
shall not we say ?
--recipient your_EVM_SC_address

Reply:
That should be implied since I use that account through this document.